tests/test_common_options.py

```
@@ -51,9 +51,9 @@ def test_list_sources_duplicates():
 # Polarion TC MTA-596
 def test_list_providers():
     kantra_path = os.getenv(constants.KANTRA_CLI_PATH)
-    command = kantra_path + ' analyze --list-providers'
+    command = kantra_path + ' provider list'
 
     output = subprocess.run(command, shell=True, check=True, stdout=subprocess.PIPE, encoding='utf-8').stdout
```

Ruff (0.15.17)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)